lib/Newsletter/Editor/PostTransformer.php

```diff
@@ -138,6 +139,25 @@ private function getContent($post, $with_post_class, $display_type) {
     return $content;
   }
 
+  private function getImageInfo($id) {
+    /*
+     * In some cases wp_get_attachment_image_src ignore the second parameter
+     * and use global variable $content_width value instead.
+     * By overriding it ourselves when ensure a constant behaviour regardless
+     * of the user setup.
+     *
+     * https://mailpoet.atlassian.net/browse/MAILPOET-1365
+     */
+    global $content_width; // default is NULL
+    
+    $content_width_copy = $content_width;
+    $content_width = Env::NEWSLETTER_CONTENT_WIDTH;
+    $image_info = $this->wp->wpGetAttachmentImageSrc($id, 'mailpoet_newsletter_max');
+    $content_width = $content_width_copy;
+
+    return $image_info;
+  }
+
```

Comment:
The commit message says `getImageInfo` was moved but I don't see it removed anywhere. What's the context/reason for this change?

Reply:
This method was part of `WP\Functions` and is only used on the `PostTransformer` class. Since I wanted the `WPFunctions` class to only contains WP functions, I moved it here.